apps/SageAccountWeb/src/index.tsx

```diff
@@ -14,7 +15,9 @@ const root = createRoot(container!)
 root.render(
   <React.StrictMode>
     <BrowserRouter>
```

Comment:
Routing is separate from the `AppWrapper` because tests will use `AppWrapper` with a `MemoryRouter`